com.stremio.Stremio.appdata.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.stremio.Stremio</id>
+  <provides><id>com.stremio.Stremio.desktop</id></provides>
+  <launchable type="desktop-id">com.stremio.Stremio.desktop</launchable>
+  <name>Stremio</name>
+  <developer_name>Stremio</developer_name>
+  <summary>Discover, organize and watch video from all kind of sources on any device that you own.</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPLv3</project_license>
+  <url type="homepage">https://www.stremio.com/</url>
+  <url type="help"></url>
```

Comment:
```suggestion
  <url type="help"></url>
```
Replace the help url with a bugtracker one, they have one over here https://github.com/Stremio/stremio-shell